templates/typescript_gapic/package.json.njk

```diff
@@ -11,15 +11,15 @@
     "!build/src/**/*.map"
   ],
   "dependencies": {
-    "google-gax": "^1.3.0"
+    "google-gax": "^1.5.0"
   },
   "devDependencies": {
     "@types/mocha": "^5.2.5",
     "@types/through2": "^2.0.34",
     "gts": "^0.9.0",
     "mocha": "^6.0.0",
     "typescript": "~3.5.3",
-    "google-gax": "^1.3.0"
+    "google-gax": "^1.5.0"
```

Comment:
`dependencies` is something that the tool needs to work, `devDependencies` is something that the tool developers need (e.g. test frameworks). Gax (its protos) are needed to work, so it should be removed from `devDependencies`: just `dependencies` is enough.